File: EjercicioC4.js
/*C4.- programa para hallar el factorial de un numero ingresado por el usuario*/ 

var factorial =1
var num=Number(prompt("Ingrese un numero: "))
for(var i=1;i<=num;i++){
    factorial=factorial*i
}

window.alert("El factorial de "+num+" es igual a : "+factorial)

File: EjercicioC4.py
"""4)programa para hallar el factorial de un numero ingresado por el usuario"""

factorial = 1
num = int(input("Ingrese un numero : "))
for i in range(1,num+1):
    factorial = factorial * i 
    
print(f"El factorial de " , num, "es igual a : ", factorial)
